codec/src/main/java/io/netty/handler/codec/json/JsonObjectDecoder.java

```diff
@@ -213,7 +213,7 @@ private void decodeByte(byte c, ByteBuf in, int idx) {
                     }
                 }
                 // The double quote isn't escaped only if there are even "\"s.
-                if (backslashCount % 2 == 0) {
```

Comment:
Have you looked what the JIT produce here?

Reply:
Yes, jit is optimized. . . Thanks for pointing out